submissions/YuliiaHordiichuk/memory-game/index.js

```diff
@@ -0,0 +1,91 @@
+class Game {
```

Comment:
Why did you decide to use a class here? As opposed to, for example, regular function

Reply:
I've got some knowledge about classes and wanted to practice it. Should I change it to a regular function?

Reply:
No, it's ok, just wanted to know the reasoning